packages/api-core/src/resources/organizations.js

```diff
@@ -45,7 +45,11 @@ export default class AvOrganizations extends AvApi {
   }
 
   async postGet(data, config, additionalPostGetArgs) {
-    if (additionalPostGetArgs) {
+    if (additionalPostGetArgs && additionalPostGetArgs.resourceIds) {
+      const { permissionIds } = additionalPostGetArgs;
+      if (permissionIds) {
+        data.permissionId = permissionIds;
```

Comment:
`data` at this point has been stringified if it's coming from a `ResourceSelect`. we need to check the `type` of data first here. if it's an object, this is fine. if it's a string, we need to `qs.parse` it, add `permissionId` and then stringify it again

https://github.com/Availity/availity-react/blob/15b8d643a239ddeb536185e5f620ce553df53ba8/packages/select/src/ResourceSelect.js#L198